### PR TITLE
docs(sense-cli): add explanation to nebula sense options

### PR DESCRIPTION
**Pull request by LiKang6688**
## Motivation

make nebula sense options clear and understandable

### CODE REVIEW

File: commands/sense/README.md [hunk 1]
```diff
@@ -45,8 +48,6 @@ export default {
 nebula sense --ext def.js
 ```
 
-The command generates all required files into the specified `--output` folder. You can the zip the folder and upload it as an extension on [Qlik Sense Enterprise for Windows](https://help.qlik.com/en-US/sense-developer/April2020/Subsystems/Extensions/Content/Sense_Extensions/Howtos/deploy-extensions.htm) or [Qlik Sense Enterprise on Kubernetes](https://help.qlik.com/en-US/sense-admin/April2020/Subsystems/DeployAdministerQSE/Content/Sense_DeployAdminister/QSEoK/Administer_QSEoK/mc-extensions.htm)
-
 ### Meta
 
 You can add more meta about the extension by providing a `.json` formatted file with `--meta`:
```
Comment (Caele) [inline commands/sense/README.md:53]: Should add an example json file here, and explain that it ends up in the qext file used by sense.

Comment (LiKang6688) [reply commands/sense/README.md:53]: I did not change this section. An example JSON file was presented before. 
Great. Gonna add that explanation about meta.

**Comment by Caele** [comment]
Looks like the merge has broken things a bit so I moved the target.

File: commands/sense/README.md [hunk 2]
```diff
@@ -62,4 +63,24 @@ You can add more meta about the extension by providing a `.json` formatted file
...
+
+### Output
+
+Generate all required files into the specified `--output` folder called sn-table-ext:
+
+```bash
+nebula sense --output sn-table-ext
+```
+
+You can upload that folder as an extension on [Qlik Sense Enterprise for Windows](https://help.qlik.com/en-US/sense-developer/April2020/Subsystems/Extensions/Content/Sense_Extensions/Howtos/deploy-extensions.htm) or [Qlik Sense Enterprise on Kubernetes](https://help.qlik.com/en-US/sense-admin/April2020/Subsystems/DeployAdministerQSE/Content/Sense_DeployAdminister/QSEoK/Administer_QSEoK/mc-extensions.htm)
```
Comment (Caele) [inline commands/sense/README.md:78]: Should update these links to the latest help.
Also switch from "on Kubernetes" to "Saas editions of Qlik Sense" https://help.qlik.com/en-US/cloud-services/Subsystems/Hub/Content/Sense_Hub/Admin/mc-extensions.htm

**Review by Caele — APPROVED** [review]
Fix the links, then we are good.

**Pull request merged**